uber/models/mits.py

```diff
@@ -166,6 +166,40 @@ class MITSGame(MagModel):
     unlicensed = Column(Boolean, default=False)
     professional = Column(Boolean, default=False)
 
+    @property
+    def has_been_accepted(self):
```

Comment:
Here's a breakdown of the MITS Game export:
We only return games from teams with an `c.ACCEPTED` status.

Name: The game's team's name.
Subtitle: The game's own name.
Description: The game's description.
Location: Nothing, but we could conceivably use MITS' location.
Image: If the team has uploaded a picture that is a 640x240, we use that. Otherwise we use the first image we find in their uploaded images (@EliAndrewC probably knows if this is deterministic or not).
Thumbnail: If the team has uploaded a picture that is 240x240, we use that. Otherwise we use the second image we find, or the first image if they only have one.

PS: The MITS team actually requested the header and thumbnail feature earlier this year specifically for Guidebook. We might even want to break this out into a utility to apply to other models, like the MIVS games.